x-pack/plugins/security_solution/scripts/junit_transformer/lib.ts

```diff
@@ -18,6 +18,20 @@ import { PathReporter } from 'io-ts/lib/PathReporter';
 import globby from 'globby';
 import del from 'del';
 
+// Function to remove specific fields from an XML object in order to
+// compare them as strings.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function removeFields(obj: any, fieldsToRemove: string[]): any {
+  for (const key in obj) {
+    if (fieldsToRemove.includes(key)) {
+      delete obj[key];
```

Comment:
NIT: I believe it would be greate if we can avoid the modification of the original object by making a copy first and doing the modifications there with the aim of avoid side effects.

Reply:
But we do not change the actual file at any point. We just getting a parsed string out of it and then we remove the fields for future use.